cli/src/installers/envVars.ts

```diff
@@ -36,42 +36,51 @@ export const envVariablesInstaller: Installer = ({ projectDir, packages }) => {
 };
 
 const getEnvContent = (usingAuth: boolean, usingPrisma: boolean) => {
-  let content =
-    "# When adding additional env variables, the schema in /env/schema.mjs should be updated accordingly";
+  let content = `
+# When adding additional environment variables, the schema in "/env/schema.mjs"
+# should be updated accordingly.
+`
+    .trim()
+    .concat("\n");
 
   if (usingPrisma)
     content += `
 # Prisma
-DATABASE_URL=file:./db.sqlite
+# https://www.prisma.io/docs/reference/database-reference/connection-urls#env
+DATABASE_URL="file:./db.sqlite"
 `;
 
   if (usingAuth)
     content += `
 # Next Auth
-# You can generate the secret via 'openssl rand -base64 32' on Linux
-# More info: https://next-auth.js.org/configuration/options#secret
-# NEXTAUTH_SECRET=
-NEXTAUTH_URL=http://localhost:3000
+# You can generate a new secret on Linux with: openssl rand -base64 32
+# https://next-auth.js.org/configuration/options#secret
+NEXTAUTH_SECRET=""
```

Comment:
This should be commented out by default since it will be handled automatically in dev